pkg/subsystem/lists/linux_upstream/subsystems.go

```diff
@@ -0,0 +1,4619 @@
+	Name:    "accessrunner",
+	Lists:   []string{"accessrunner-general@lists.sourceforge.net"},
+	Parents: []*Subsystem{kernel, usb},
+	PathRules: []PathRule{
+		{IncludeRegexp: "^drivers/usb/atm/cxacru\\.c$"},
+	},
+}
+
+var acenic = &Subsystem{
+	Name:    "acenic",
+	Lists:   []string{"linux-acenic@sunsite.dk"},
+	Parents: []*Subsystem{net},
+	PathRules: []PathRule{
+		{IncludeRegexp: "^drivers/net/ethernet/alteon/acenic(?:[^/]*)$"},
+	},
+}
+
+var acpi = &Subsystem{
+	Name:    "acpi",
+	Lists:   []string{"linux-acpi@vger.kernel.org"},
+	Parents: []*Subsystem{kernel},
+	PathRules: []PathRule{
+		{IncludeRegexp: "^drivers/acpi(?:/.*)?$|^drivers/pci/(?:[^/]*)/(?:[^/]*)acpi(?:[^/]*)$|^drivers/pci/(?:[^/]*)acpi(?:[^/]*)$|^drivers/pnp/pnpacpi(?:/.*)?$|^include/acpi(?:/.*)?$|^include/linux/acpi\\.h$|^include/linux/fwnode\\.h$"},
+		{IncludeRegexp: "^drivers/acpi/(?:[^/]*)thermal(?:[^/]*)$"},
+		{IncludeRegexp: "^drivers/acpi/acpica(?:/.*)?$|^include/acpi(?:/.*)?$"},
+		{IncludeRegexp: "^drivers/acpi/apei(?:/.*)?$"},
+		{IncludeRegexp: "^drivers/acpi/arm64$"},
+		{IncludeRegexp: "^drivers/acpi/pmic(?:/.*)?$"},
+		{IncludeRegexp: "^drivers/acpi/viot\\.c$|^include/linux/acpi_viot\\.h$"},
+		{IncludeRegexp: "^drivers/base/property\\.c$|^drivers/base/swnode\\.c$|^include/linux/fwnode\\.h$|^include/linux/property\\.h$"},
+		{IncludeRegexp: "^drivers/gpio/gpiolib-acpi\\.c$|^drivers/gpio/gpiolib-acpi\\.h$"},
+		{IncludeRegexp: "^drivers/i2c/i2c-core-acpi\\.c$"},
+		{IncludeRegexp: "^drivers/mailbox/pcc\\.c$"},
+		{IncludeRegexp: "^drivers/pnp(?:/.*)?$|^include/linux/pnp\\.h$"},
+	},
+}
+
+var acrn = &Subsystem{
+	Name:    "acrn",
+	Lists:   []string{"acrn-dev@lists.projectacrn.org"},
+	Parents: []*Subsystem{kernel},
+	PathRules: []PathRule{
+		{IncludeRegexp: "^drivers/virt/acrn(?:/.*)?$|^include/uapi/linux/acrn\\.h$"},
+	},
+}
+
+var actions = &Subsystem{
+	Name:    "actions",
+	Lists:   []string{"linux-actions@lists.infradead.org"},
+	Parents: []*Subsystem{kernel},
+	PathRules: []PathRule{
+		{IncludeRegexp: "^arch/arm/boot/dts/owl-(?:[^/]*)$|^arch/arm/mach-actions(?:/.*)?$|^arch/arm64/boot/dts/actions(?:/.*)?$|^drivers/clk/actions(?:/.*)?$|^drivers/clocksource/timer-owl(?:[^/]*)$|^drivers/dma/owl-dma\\.c$|^drivers/i2c/busses/i2c-owl\\.c$|^drivers/irqchip/irq-owl-sirq\\.c$|^drivers/mmc/host/owl-mmc\\.c$|^drivers/net/ethernet/actions(?:/.*)?$|^drivers/pinctrl/actions/(?:[^/]*)$|^drivers/soc/actions(?:/.*)?$|^include/dt-bindings/power/owl-(?:[^/]*)$|^include/dt-bindings/reset/actions,(?:[^/]*)$|^include/linux/soc/actions(?:/.*)?$|owl"},
+		{IncludeRegexp: "^drivers/input/misc/atc260x-onkey\\.c$|^drivers/mfd/atc260(?:[^/]*)$|^drivers/power/reset/atc260x-poweroff\\.c$|^drivers/regulator/atc260x-regulator\\.c$|^include/linux/mfd/atc260x/(?:[^/]*)$"},
+	},
+}
+
+var adfs = &Subsystem{
+	Name:      "adfs",
+	Syscalls:  []string{"syz_mount_image$adfs"},
+	Lists:     []string(nil),
+	Parents:   []*Subsystem{},
+	PathRules: []PathRule(nil),
```

Comment:
The serializer would not emit Lists,  Parents, PathRules for empty fields.
It's the side-effect of separate field serialization. We could export `serializer.isDefaultValue` to skip serialization. But using strings to refer to parents would solve this as well.

Reply:
In the current version the generating code does not print those lines if they are empty.